index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
+	return '$m_' + str.replace(/-(\w)/g, (_, letter) => letter.toUpperCase()).replace(/[^$_a-zA-Z0-9]/g, '_');
+}
+
+function writePrefix (imports) {
+	let transformed = `Object.defineProperty(exports, '__esModule', { value: true });\n`;
```

Comment:
I don't think we can avoid the commonjs interop. I think a route like Babel's will be good. Folks don't want to think about the module format of what they're consuming.